server/lib/validation/languages/yue.js

```diff
@@ -0,0 +1,34 @@
+// Minimum of words that qualify as a sentence.
+const MIN_LENGTH = 3;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
```

Comment:
minor: could you please change the comment here as it's now number of characters and not words?